daemon/src/rollover_maker.rs

```diff
@@ -40,6 +41,7 @@ pub struct ProtocolMsg(pub wire::RolloverMsg);
 /// notify that rollover has finished successfully.
 struct RolloverSucceeded {
     dlc: Dlc,
+    funding_fee: FundingFee,
 }
 
 /// Message sent from the spawned task to `rollover_taker::Actor` to
```

Comment:
```suggestion
/// Message sent from the spawned task to `rollover_maker::Actor` to
```

Just noticed that this comment is confusing, I think it should be the maker here...?